src/v/kafka/protocol/tests/protocol_test.cc

```diff
@@ -12,6 +12,7 @@
 
 #include <boost/process.hpp>
 #include <boost/test/unit_test.hpp>
+#include <fmt/format.h>
```

Comment:
is this needed?

Reply:
It's not necessary for this PR. I removed it now.